06_PDDE_Checker.R

```diff
@@ -71,8 +71,13 @@ operating_end_missing <- Enrollment %>%
 
 # Missing CoC Information Missing address field(s), Missing Geocode,
 # Missing Geography Type, Invalid Zip Code if possible
+
+HousingTypeDF <- Project %>% 
+  subset(select=c(ProjectID, HousingType))
```

Comment:
a more consistent-with-existing-code approach is to just do:

```
HousingTypeDF <- Project %>%
  select(ProjectID, HousingType)
```

Reply:
Edit made